pangeo_forge_recipes/transforms.py

```diff
@@ -409,6 +404,25 @@ def expand(self, pcoll: beam.PCollection) -> beam.PCollection:
         return new_fragments
 
 
+def _consolidate_zarr_metadata(store: zarr.storage.FSStore) -> zarr.storage.FSStore:
```

Comment:
this will work for Kerchunk as well, but the typehint needs to be `MutableMapping` here